app/.gitignore

```diff
@@ -1 +0,0 @@
-/build
```

Comment:
Oh wat put that back in. I was wondering why

On Sat, Jan 9, 2016, 11:46 PM Austin Chen <notifications@github.com> wrote:

> In app/.gitignore
> <https://github.com/williamxu/MangoReader/pull/79#discussion_r49272797>:
>
> > @@ -1 +0,0 @@
> > -/build
>
> Did you do this on purpose? Without this .gitignore a lot of temporary
> build files show up...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/williamxu/MangoReader/pull/79/files#r49272797>.
>